rbc/heavydb/remoteheavydb.py

```diff
@@ -1508,5 +1508,5 @@ def remote_call(self, func, ftype: typesystem.Type, arguments: tuple, hold=False
 
 
 class RemoteOmnisci(RemoteHeavyDB):
-    """Omnisci - the previous brand of HeavyAI
+    """HeavyDB - the previous brand of HeavyAI
```

Comment:
This is actually Omnisci:
```suggestion
    """Omnisci - the previous brand of HeavyAI
```
However, as soon as HeavyDB 6.2 is out, we can drop Omnisci support.